docs/conf.py

```diff
@@ -25,9 +25,6 @@
 # Import for custom theme from Read the Docs
 import sphinx_rtd_theme
 
-sys.path.insert(0, os.path.abspath('..'))
```

Comment:
I think you need to delete the import of `os` now that it is unused to make `flake8` happy